_inc/client/state/site-verify/reducer.js

```diff
@@ -90,3 +94,15 @@ export function isConnectedToGoogleSiteVerificationAPI( state ) {
 export function isSiteVerifiedWithGoogle( state ) {
 	return get( state, 'jetpack.siteVerify.google.verified', false );
 }
+
+export function getGoogleSiteVerificationError( state ) {
+	return get( state, 'jetpack.siteVerify.google.error', null );
+}
+
+export function getGoogleSearchConsoleUrl( state ) {
+	return get( state, 'jetpack.siteVerify.google.google_search_console_url', null );
+}
+
+export function getGoogleVerificationConsoleUrl( state ) {
+	return get( state, 'jetpack.siteVerify.google.google_verification_console_url', null );
```

Comment:
Super minor, and absolutely not mandatory, but we could ditch the `google` part in property names since it's both repetitive and longer:

```javascript
return get( state, 'jetpack.siteVerify.google.verification_console_url', null );
```